Translation-Chinese.md

```diff
@@ -6,62 +6,64 @@
 [Русский](./Translation-Russian.md) ·
 [Deutsch](./Translation-German.md) ·
 [日本語](./Translation-Japanese.md)
+[Polski](./Translation-Polish.md)
 
 ---
 
 # 学习 LaTeX 从现在开始
 
 ![](https://upload.wikimedia.org/wikipedia/commons/9/92/LaTeX_logo.svg)
 
-**译者前言：** *本人尽量按照原作者原意翻译，但水平有限，如有翻译不妥之处，欢迎指正。如果您喜欢本文，欢迎分享给更多人看到。*
```

Comment:
What's happening here? Were you using some sort of WYSIWYG Markdown editor which prefers `_` over `*`?

Reply:
Probably it's effect of VSCode format-on-save feature.

Reply:
Restored